mongoid.gemspec

```diff
@@ -42,7 +42,7 @@ Gem::Specification.new do |s|
   s.add_dependency("mongo", ['>=2.18.0', '<3.0.0'])
   s.add_dependency("concurrent-ruby", ['>= 1.0.5', '< 2.0'])
 
-  s.add_development_dependency("bson", ['>=4.14.0', '<5.0.0'])
+  s.add_development_dependency("bson", '>=4.14.0')
```

Comment:
'< 6.0.0' is prudent here, unless you can absolutely guarantee that all future version of BSON will be compatible with this version of Mongoid.